README.md

```diff
@@ -260,13 +272,22 @@ spark-submit \
   --job-name "demo_photon_ml_logistic_regression"
 ```
 
-There is also a more complex script demonstrating advanced options and customizations of using Photon ML at  [example/run_photon_ml.driver.sh](https://github.com/linkedin/photon-ml/blob/master/examples/run_photon_ml_driver.sh).
+There is also a more complex script demonstrating advanced options and customizations of using Photon ML at [example/run_photon_ml.driver.sh](https://github.com/linkedin/photon-ml/blob/master/examples/run_photon_ml_driver.sh).
 
 Detailed usages are described via command:
 ```bash
 ./run_photon_ml.driver.sh [-h|--help]
 ```
 **Note**: not all configurations are currently exposed as options in the current script, please directly modify the confs if any customization is needed.
 
+#### Running Photon ML on Other Cluster Mode
+Similar to running Photon ML or other general Spark applications on YARN, running Photon ML on other cluster mode (e.g.
+[Spark Standalone Mode](http://spark.apache.org/docs/latest/spark-standalone.html)
+and [Mesos](http://spark.apache.org/docs/latest/running-on-mesos.html)) can be done by modifying the Spark-specific
+arguments of the above example scripts accordingly as detailed in
+[http://spark.apache.org/docs/latest/cluster-overview.html](http://spark.apache.org/docs/latest/cluster-overview.html).
+For example, changing ```--master``` and ```--deploy-model``` and related Spark-specific arguments to cope with
```

Comment:
Should be "deploy-mode". As I understand it, 'master' and 'deploy-mode' are the only two arguments that affect where a Spark application is run. Writing "related Spark-specific arguments" muddies this, since the other Spark-specific arguments are for how it runs the application, not where.

We should also have a link to this page somewhere in here: http://spark.apache.org/docs/latest/submitting-applications.html